node/src/components/fetcher.rs

```diff
@@ -151,18 +151,14 @@ pub(crate) trait ItemFetcher<T: Item + 'static> {
         id: T::Id,
         peer: NodeId,
     ) -> Effects<Event<T>> {
+        let peer_timeout = self.peer_timeout();
```

Comment:
If I wanted to be pedantic, I could say we can move this inside the `Ok` branch. But we will never trigger the `Err` anway :P

Reply:
Not

```rust
 Ok(message) => async move {
                let peer_timeout = self.peer_timeout();
                effect_builder.send_message(peer, message).await;

                effect_builder.set_timeout(peer_timeout).await
```

but

```rust
 Ok(message) => {
        let peer_timeout = self.peer_timeout();
        async move {
                effect_builder.send_message(peer, message).await;

                effect_builder.set_timeout(peer_timeout).await
```

Reply:
But let's keep as is, easier on the eyes :)